pyscripts/inject_into_junit.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for
+# full license information.
+#
+# filename: inject_into_junit.py
+# author:   v-greach@microsoft.com
+# created:  03/11/2019
+# Rev: 03/11/2019 D
+
+import sys
+import os
+import argparse
+import shutil
+
+class InjectIntoJunit:
+
+    def __init__(self, args):
+
+        # Parse args
+        parser = argparse.ArgumentParser(description="Inject Into Junit")
+        parser.add_argument('-junit_file', required=True, nargs=1, help="filename of junit file")
+        parser.add_argument('-log_file', required=True, nargs=1, help="filename of log file")
+        parser.add_argument('-whatif', action="store_true", default=False, help="for testing")
```

Comment:
>parser.add_argument('-whatif', action="store_true", default=False, help="for testing") [](start = 8, length = 86)

Either remove this or explain it better please.

Reply:
Removed -whatif